app/routes/articles.js

```diff
@@ -133,12 +132,12 @@ router.post('/listings', listingsLimit, (req, res) => {
         catch(err => req.log('articleListingPromise:', err))
 })
 
-//GET Article Images
+
 router.get('/image/:link', async (req, res) => {
     if(!req.params.link) return res.sendStatus(404)
     const {width, height} = req.query
-    if(width && height) return getArticleImage(res, req.params.link, width, height).catch(err => req.log(err))
-    return getArticleImage(res, req.params.link).catch(err => req.log(err))
+    const article = await Article.findOne({link: req.params.link}).exec()
+    return getArticleImage(res, article?.headerUrl || req.params.link, width, height)
```

Comment:
If the link provided is not the title of an article, it's either a content image or it doesn't exist. Hence why I'm still propagating req.params.link